client/admin-pointers.js

```diff
@@ -37,11 +35,35 @@ jQuery( document ).ready( function( $ ) {
 			},
 		} );
 
-		target.pointer( options ).pointer( 'open' );
-		if ( pointer.dim ) {
-			target.css( 'z-index', 9999 );
-			$( 'body' ).append( '<div id="wcs-pointer-page-dimmer" class="wcs-pointer-page-dimmer"></div>' );
-			$( '#wcs-pointer-page-dimmer' ).fadeIn( 500 );
+		function open() {
+			const target = $( pointer.target );
+			target.pointer( options ).pointer( 'open' );
+			if ( pointer.dim ) {
+				target.css( 'z-index', 9999 );
+				$( 'body' ).append( '<div id="wcs-pointer-page-dimmer" class="wcs-pointer-page-dimmer"></div>' );
+				$( '#wcs-pointer-page-dimmer' ).fadeIn( 500 );
+			}
+		}
+
+		// Selectors for delayed opening. Only show_button is required.
```

Comment:
Can you expand this comment to describe the uses of all of the `delayed_opening` parameters? _(e.g. static vs. dynamic containers)_